### ISSUE TITLE
changing widgets always jumps to system info widet of settings

### ISSUE THREAD

**Issue by mgonzales71** [marcelveldt/skin.titan#213]
not sure when this issue first appeared in the skin but i am on jarvis beta 3.6.63 of skin, 1.0.92 of helper and 1.0.9 of skin shortcuts under 16.1 of Kodi.

basically when you try to cycle / change a widget the skin jumps to system info widget of settings home tile - this happens for all multi-widget home tiles that I have.

vid of behaviour:
[2016-05-16_14-42-14-Segment 1.zip](https://github.com/marcelveldt/skin.titan/files/267036/2016-05-16_14-42-14-Segment.1.zip)

**Comment by marcelveldt** [comment]
Hi Mario, this should be fixed in recent version of the skin. Can you confirm ?

**Comment by marcelveldt** [comment]
@mgonzales71 assuming this is fixed for now. If not, please reopen.

**Comment by mgonzales71** [comment]
I actually saw it this weekend but I can't reproduce it now that I have moved to Kodi Krypton Alpha and the Krypton Titan Beta skin.  

just lots of  windows translation errors in the skin - 
11:21:24 T:9580   ERROR: Window Translator: Can't find window mutebug
11:21:24 T:9580   ERROR: Window Translator: Can't find window musiclibrary
(and other windows)

a reoccuring include error with I am guessing widgets/skin shortcuts:

11:20:59 T:376   ERROR: XFILE::CDirectory::GetDirectory - Error getting $INCLUDE[SystemInfoWidget]

This curious entry:

10:57:49 T:9580 WARNING: JSONRPC: Could not parse type "Setting.Details.SettingList"
10:57:49 T:9580 WARNING: CApplication::Initialize - startup.xml taints init process

and twice while playing with other skins in Krypton when I went to switch to Titan from another skin the skin helper service's locahost web service died or did not start (lots errors) a restart of the Kodi app fixed it but have not yet caught a debug log of this yet - will let you know if I do.

Thanks Marcel!

Mario

I know, neither Kodi or the Skin are there yet so I not to worried.
